.c8rc.json

```diff
@@ -1,4 +1,5 @@
 {
-  "reporter": ["html", "lcov"],
-  "exclude": ["test/*", "*.mjs", "**/*.mjs"]
+  "reporter": ["lcov"],
+  "exclude": ["test/*", "*.mjs", "**/*.mjs"],
+  "clean": true
```

Comment:
this cleans the coverage json files in tmp dir, only really meant for local testing